app/assets/javascripts/modules/track-submit.js

```diff
@@ -3,18 +3,19 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 (function (Modules) {
   'use strict'
 
-  Modules.TrackSubmit = function () {
-    this.start = function (element) {
-      element.on('submit', 'form', trackSubmit)
+  Modules.TrackSubmit = function (element) {
+    this.$module = element
+    this.formElement = this.$module.querySelector('form')
+  }
 
-      var category = element.data('track-category')
-      var action = element.data('track-action')
+  Modules.TrackSubmit.prototype.init = function () {
+    var category = this.$module.getAttribute('data-track-category')
```

Comment:
I've just thought... in the world of jQuery if this module were applied to something that didn't contain a form, it would just fail silently. Here, that won't happen. It might be worth wrapping the entire contents of the `init` function in a `if (this.formElement) {` to prevent such an error from occurring.